server/tests/test_notebook_revisions_cleanup.py

```diff
@@ -38,13 +43,29 @@ def notebook_and_draft_revisions(fake_user):
         )
         revision.created = created
         revision.save()
-        notebook_revisions.append(revision)
 
-    return notebook, notebook_revisions
+    return notebook
+
+
+@pytest.fixture
+def notebook_with_duplicate_draft_revisions(fake_user):
+    notebook = Notebook.objects.create(owner=fake_user, title="Fake notebook")
+
+    for i, created in enumerate(CREATED_DATETIMES_FOR_DUPLICATE_REVISIONS):
+        revision = NotebookRevision.objects.create(
+            notebook=notebook,
+            title=f"Revision",
```

Comment:
Format string not needed

```suggestion
            title="Revision",
```